src/pytesmo/metrics.py

```diff
@@ -257,7 +256,7 @@ def tcol_error(x, y, z):
 
     return e_x, e_y, e_z
 
-
+@np.errstate(invalid='ignore')
```

Comment:
- Some other warnings came from numpy and are due to invalid operations (div0, sqrt(<0) etc.), I have now added a flag to exclude these warnings when in some places